### DISCUSSION TITLE
如何修改日历显示的行数

### DISCUSSION THREAD

**Discussion by hundanwan** [ant-design/ant-design#37607] [Q&A]
比如现在显示的是六行，一行是一周，但是我现在想要只展示五行要怎么修改

**Comment by fnoopv** [comment]
很好奇，什么样的需求才需要这样的功能设计？

**Reply by hundanwan** [reply]
设计稿子上要求的(哭

**Reply by fnoopv** [reply]
这样的需求好像挺难的，`antd`的`DatePicker`是基于[picker](https://github.com/react-component/picker)实现的，这里面有一系列功能和样式设计，多一行或少一行就会导致原来的功能出问题了，比如选择一周时的样式显示什么的。或者你可以基于[picker](https://github.com/react-component/picker)魔改一版？

**Reply by hundanwan** [reply]
谢谢，这个问题出来的第二天已经和设计进行 友好交流 然后解决了

**Reply by fnoopv** [reply]
哈哈，这才是解决方法
